test/CMakeLists.txt

```diff
@@ -0,0 +1,917 @@
+set(ZFP_LIB_VERSION ${ZFP_VERSION_MAJOR}${ZFP_VERSION_MINOR}${ZFP_VERSION_PATCH})
+if (ZFP_VERSION VERSION_GREATER_EQUAL 1.0.0)
+  set(ZFP_LIB_VERSION ${ZFP_LIB_VERSION}${ZFP_VERSION_TWEAK})
+endif ()
+set(ZFP_HAS_CFP 0)
+if (ZFP_CFP_ENABLED)
+  set(ZFP_HAS_CFP 1)
+endif()
+
+#------------------------------------------------------------------------------#
+# Custom defined HDF5 variables
+#------------------------------------------------------------------------------#
+get_filename_component(HDF5_BIN_DIR ${HDF5_DIFF_EXECUTABLE} DIRECTORY)
+set(HDF5_REPACK_EXECUTABLE ${HDF5_BIN_DIR}/h5repack)
+if (EXISTS ${HDF5_BIN_DIR}/h5repack-shared)
+  set(HDF5_REPACK_EXECUTABLE ${HDF5_BIN_DIR}/h5repack-shared)
+endif ()
+set(HDF5_DUMP_EXECUTABLE ${HDF5_BIN_DIR}/h5dump)
+
+#------------------------------------------------------------------------------#
+# Setup HDF5_PLUGIN_PATH
+#------------------------------------------------------------------------------#
+get_target_property(HDF5_PLUGIN_PATH h5z_zfp_shared RUNTIME_OUTPUT_DIRECTORY)
+if (_isMultiConfig)
+  set(HDF5_PLUGIN_PATH "${HDF5_PLUGIN_PATH}/$<CONFIG>")
+endif ()
+
+#------------------------------------------------------------------------------#
+# Plugin and library write & read tests (No check implemented)
+#------------------------------------------------------------------------------#
+add_executable(test_write_plugin test_write.c)
+target_compile_definitions(test_write_plugin PRIVATE H5Z_ZFP_USE_PLUGIN)
+target_compile_definitions(test_write_plugin PRIVATE ZFP_LIB_VERSION=0x${ZFP_LIB_VERSION})
+target_link_libraries(test_write_plugin h5z_zfp_shared)
+
+add_executable(test_read_plugin test_read.c)
+target_compile_definitions(test_read_plugin PRIVATE H5Z_ZFP_USE_PLUGIN)
+target_link_libraries(test_read_plugin h5z_zfp_shared)
+
+add_executable(test_write_lib test_write.c)
+target_compile_definitions(test_write_lib PRIVATE ZFP_LIB_VERSION=0x${ZFP_LIB_VERSION})
+target_compile_definitions(test_write_lib PRIVATE ZFP_HAS_CFP=${ZFP_HAS_CFP})
+target_link_libraries(test_write_lib h5z_zfp_static)
+if (ZFP_CFP_ENABLED)
+  target_link_libraries(test_write_lib zfp::cfp)
+endif ()
+
+add_executable(test_read_lib test_read.c)
+target_link_libraries(test_read_lib h5z_zfp_static)
+
+if (FORTRAN_INTERFACE)
+  add_executable(test_rw_fortran test_rw_fortran.F90)
+  target_link_libraries(test_rw_fortran h5z_zfp_static)
+#<<<<<<< HEAD
```

Comment:
This appears to be a diff/merge output commented out. Should it remain here?

Reply:
fixed.